### DISCUSSION TITLE
Add option to copy git/repo link to clipboard on sharing options

### DISCUSSION THREAD

**Discussion by rBok6** [community/community#15161] [Mobile]
It's impossible to copy a git/repo link to the clipboard on the mobile app.

**Comment by haldun** [comment]
Hi @rBok6, thank you for the feedback. Can you please share whether you're using iOS or the Android app?

**Reply by rBok6** [reply]
I'm using Android 10.

**Reply by haldun** [reply]
Thanks! There is a copy option if you tap on the three dots and select "Share" menu item in the repository detail screen.

**Reply by rBok6** [reply]
I'm sorry but I don't seem to have that option and I'm using the latest version from Play store.

![IMG_20220419_102302.jpg](https://user-images.githubusercontent.com/13161094/164027056-5558276d-caf3-4e00-8e4d-129d24e0dcb1.jpg)
![IMG_20220419_102533.jpg](https://user-images.githubusercontent.com/13161094/164027088-f6e7a594-6bec-4fad-8e5e-9d925030be50.jpg)

**Reply by haldun** [reply]
Thanks a lot for the screenshots, I am filing this and report to the team.

**Reply by rBok6** [reply]
Thank you.

**Reply by stevepopovich** [reply]
Since this should be fixed in newer versions of Android, we recommend updating your operating system. We won't be taking any direct action. Thanks!

**Reply by rBok6** [reply]
What do you consider newer Android? Android 10 isn't newer enough or what? I updated to the latest version and the issue still persists. How difficult can it be to add an option to copy the repo link directly to the clipboard? 

So, is it ok to select an unwanted app to share the link to, just to reveal the repo link?

Nice user experience there... 😒

**Reply by alcere** [reply]
Hi @rBok6, we are sorry for the inconvenience :(. We will re-open the issue and try to find a solution; sadly, it won't be happening in the short term as it depends on the ROM customization from the phone manufacturer, but we will take another look and evaluate your suggestion with the design team. Thanks again!

**Comment by rBok6** [comment]
I'm closing this issues because the "copy repo link" option is now working since I upgraded my phone's system to Android 11. But I'm kind of disappointed that such a small feature had to be ROM dependant and makes awful the user experience on Android 10.
